optimum/intel/openvino/modeling_base.py

```diff
@@ -434,6 +491,14 @@ def _from_transformers(
         # would end-up removing the directory containing the underlying OpenVINO model
         cls._model_save_dir_tempdirectory_instance = save_dir
 
+        compile_only = kwargs.pop("compile_only", False)
+        if compile_only:
+            logger.warning(
+                "`compile_only` mode will be disabled because it does not support model export."
+                "Please provide openvino model obtained using optimum-cli or saved on disk using `save_pretrained`"
+            )
+            compile_only = False
```

Comment:
same thing, dismissing an explicit argument with a warning doesn't make sense to me